src/app/views/query-runner/QueryRunner.tsx

```diff
@@ -85,10 +80,23 @@ export class QueryRunner extends Component<
     }
   };
 
-  public render() {
-    const { httpMethods } = this.state;
-    const { graphExplorerMode, isLoadingData } = this.props;
+  private handleOnVersionChange = (option?: IDropdownOption) => {
+    const { sampleQuery } = this.props;
+    if (option) {
+      const urlObject: URL = new URL(sampleQuery.sampleUrl);
+      const requestUrl = urlObject.pathname.substr(5).replace(/\/$/, '');
+      const sampleUrl = `${GRAPH_URL}/${option.text + requestUrl + decodeURI(urlObject.search)}`;
+
+      this.props.actions!.selectQueryVersion(option.text);
```

Comment:
This action creator sets the query version. A more appropriate name will be setQueryVersion.